mlir/test/L0_RUNNER/lit.local.cfg

```diff
@@ -1 +1,6 @@
-config.unsupported = True
\ No newline at end of file
+import os
+
+if os.environ.get('IMEX_ENABLE_GPU_E2E_TEST') == "1":
+    config.unsupported = False
+else:
+    config.unsupported = True
```

Comment:
`config.unsupported = bool(int(os.environ.get('IMEX_ENABLE_GPU_E2E_TEST', 0)))`

Reply:
Also, rename it to 'IMEX_ENABLE_GPU_E2E_TESTS'

Reply:
OK, done

Reply:
I forgot `not` :)

Reply:
Haha, I see